src/store/modules/ProfilesModule.ts

```diff
@@ -92,10 +95,11 @@ export const ProfilesModule = {
 
         async updateProfileList({commit, rootGetters}) {
             const profilesDirectory = Profile.getRootDir();
-
+            await FileUtils.ensureDirectory(profilesDirectory);
             let profilesDirectoryContents = await FsProvider.instance.readdir(profilesDirectory);
             let promises = profilesDirectoryContents.map(async function(file) {
-                return ((await FsProvider.instance.stat(path.join(profilesDirectory, file))).isDirectory() && file.toLowerCase() !== 'default' && file.toLowerCase() !== "_profile_update")
+                const fileStat = await FsProvider.instance.stat(path.join(profilesDirectory, file));
+                return (fileStat.isDirectory() && file.toLowerCase() !== 'default' && file.toLowerCase() !== "_profile_update")
```

Comment:
Is this the sort of fileStat where `isDirectory` method was overwritten with the boolean value? Or do we have multiple FS implementations? Does store modules have access to node methods without having to go through IPC? Where do the boundaries between the renderer and main process actually stand? 😕